src/cmd/manage_nsfs.js

```diff
@@ -395,111 +389,75 @@ async function fetch_existing_account_data(action, target, decrypt_secret_key) {
     return data;
 }
 
+/**
+ * add_account creates a new account config file, 
+ * @param {Object} data
+ * @returns { Promise<{ code: typeof ManageCLIResponse.AccountCreated, detail: Object, event_arg: Object  }>} 
+ */
 async function add_account(data) {
-    await manage_nsfs_validations.validate_account_args(config_fs, data, ACTIONS.ADD, undefined);
-
-    const access_key = has_access_keys(data.access_keys) ? data.access_keys[0].access_key : undefined;
-    const name_exists = await config_fs.is_account_exists_by_name(data.name);
-    const access_key_exists = access_key && await config_fs.is_account_exists_by_access_key(access_key);
-
-    const event_arg = data.name ? data.name : access_key;
-    if (name_exists || access_key_exists) {
-        const err_code = name_exists ? ManageCLIError.AccountNameAlreadyExists : ManageCLIError.AccountAccessKeyAlreadyExists;
-        throw_cli_error(err_code, event_arg, {account: event_arg});
-    }
     data._id = mongo_utils.mongoObjectId();
```

Comment:
I think this line can also move to `config_fs.create_account_config_file`, as the module manages the config directory it should be responsible for attaching the ID for a created config.